getting-started.md

```diff
@@ -1,13 +1,19 @@
 ---
-title: Getting Started
+title: Getting Started with BDI
 category: 2. Getting Started
 order: 2
 ---
 
-Getting started with BDI
+To get started with BDI, there are a few things you need to acquire or install.
 
-What do you need to get started?
-Certificates, account, access keys
-What do you have to implement to get started? As which role?
-What does a typical network setup look like?
-Does that apply to both production and development? (no, this site is meant to help development)
+First, whether you are developing a Service Consumer or a Service Provider, you will need to be registered by an Association Register in the Data Space where you want to work. You will need an EORI number and credentials, including a private key and a certificate. You will also need to know the endpoint of the Association Register. If you are building a Service Consumer, you will also need to know the EORI numbers and endpoints of all Service Providers you plan to use.
+
+For this project, we assume you will be using JavaScript. We have created cookbook recipes in JavaScript for several scenarios, both for Service Consumer and Service Provider scenarios. You can find links to them in the sidebar on the left.
```

Comment:
"For this project" ?

Misschien is het duidelijker als we de stukken die specifiek over requirements van de cookbooks gaan bij de cookbooks plaatsen en dit hier net anders doen bijvoorbeeld

"We have provided cookbooks for common scenario's. These are written in javascript <link naar cookbooks>. If you prefer to use another language, that is possible too." ....